packages/ubuntu_bootstrap/lib/installer.dart

```diff
@@ -53,8 +52,6 @@ Future<void> runInstallerApp(
         defaultsTo: 'examples/sources/desktop.yaml',
         help: 'Path of the source catalog (dry-run only)');
     parser.addFlag('welcome', aliases: ['try-or-install'], hide: true);
-    parser.addOption('pages',
-        valueHelp: 'path', help: 'A comma-separated list of pages', hide: true);
```

Comment:
Were we going to drop command line support completely?

Reply:
I don't think it makes sense to have a command line option that allows to *include* specific pages, while we're using the config entries to *exclude* pages.
I'd come back to command line options at a later point - maybe we'll want to have both `--exclude-pages` and `--include-pages` or something.